src/components/ButtonWraper.jsx

```diff
@@ -6,19 +6,13 @@
 /* eslint-disable linebreak-style */
 import './ButtonWraper.css';
 import React, { Component } from 'react';
-import Button from './Button';
+// import Button from './Button';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class ButtonWraper extends Component {
   render() {
-    const numbers = ['A/C', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', ' 2', '3', '+', '0', '.', '='];
-
-    // changeCount();
-
     return (
-      <div className="buttonBox">
-        {numbers && numbers.map((number) => <Button number={number} key={number} handleClick={this.props.handleClick} />)}
-      </div>
+      <div className="buttonBox">{this.props.children}</div>
     );
   }
 }

```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.